initial/pom.xml

```diff
@@ -28,6 +28,7 @@
 			<groupId>org.springframework.boot</groupId>
 			<artifactId>spring-boot-starter-test</artifactId>
 			<scope>test</scope>
+			<!-- We'll be using JUnit 5. Vintage is for JUnit 4 so we exclude it. -->
 			<exclusions>
```

Comment:
Excluding vintage seems like a good idea, but it doesn't come along with Spring Initializr. We just need to add a part in the tutorial to add this exclusion.

Reply:
Better simply to omit this part - it shouldn't be necessary with latest version of Spring Boot.

Reply:
I think the `build.gradle` needs to be fixed though.

Reply:
@dsyer, I've got a pull request aimed at starting this tutorial using the Spring Initializr (#11). That should fix any build concerns.

What do you think about that course of action?

Reply:
So this PR is redundant?

Reply:
@dsyer, fixing the build in both places would be redundant, but no the PR is not redundant. This PR was mostly about fixing wording issues and adding clarifications; the other PR is focused on starting using Initializr.

Reply:
OK, but the exclusion is redundant still? I mean there’s something that needs fixing, I guess I don’t care which PR it happens in.

Reply:
It looks like [after Spring Boot 2.4, the exclusion is redundant](https://github.com/spring-projects/spring-boot/wiki/Spring-Boot-2.4-Release-Notes#junit-5s-vintage-engine-removed-from-spring-boot-starter-test).

But, in this PR the version is still 2.3.2.RELEASE, so excluding vintage here is not redundant. 

In #11, we move this tutorial to the latest version from Initialzr which is 2.4.1; but since we'll then be using the POM and other build files provided by Initializr, there's no problem there.

Reply:
So I was mistaken earlier when I said

> We just need to add a part in the tutorial to add this exclusion.

Adding the exclusion isn't necessary with 2.4.1 and if you happen to be using an older version from Initialzr, it will exclude it if necessary.